src/services/AbstractDFG.jl

```diff
@@ -52,6 +52,13 @@ getAddHistory(dfg::AbstractDFG) = dfg.addHistory
 """
 getSolverParams(dfg::AbstractDFG) = dfg.solverParams
 
+"""
+    $(SIGNATURES)
+"""
+getFactorOperationalMemoryType(par::AbstractParams) = par.factorOperationalMemoryType
+
+getFactorOperationalMemoryType(dfg::AbstractDFG) = getFactorOperationalMemoryType(getSolverParams(dfg))
```

Comment:
I think we might be missing each other here, we are not currently storing `ConvolutionalObject` so why would you need to store `FactorOperationalMemory`?  Should it not just be the same abstracts but better names and some of the IIF functionality moves up to DFG as needed?  My sense is quite against storing Type information in this way but im probably missing something?

Reply:
okay so you did that in #422 , but then im missing what the idea is here?  I'm still not sure we should be storing Type information as currently suggested.

Reply:
also see: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/pull/433#discussion_r419142906

Reply:
right so, trying to do ... ? https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/425#issuecomment-623853861

Reply:
Main idea is to make DFG not depend on any downstream functions to make testing and maintenance easier.

Reply:
I'm open for other suggestions with the restraint that it is testable in DFG without needing IIF.